rke/k8s_rke_system_images.go

```diff
@@ -2438,6 +2438,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.3"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled out of band after v2.4.3
+		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
+		"v1.16.11-rancher1-1": {
+			Etcd:                      m("quay.io/coreos/etcd:v3.3.15-rancher1"),
+			Kubernetes:                m("rancher/hyperkube:v1.16.11-rancher1"),
```

Comment:
This image doesn't exist yet, right? Can we make this `v1.16.10-rancher2`?

Reply:
Yea, I think I misunderstood this process. I thought we weren't merging until the cve is released, so yea those images don't exist, they were just placeholders for what will be the new hyperkube image. I'll change it